CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 - Added convenience method to set properties for all `MessageSizeCalculator`s.
 [#697](https://github.com/MessageKit/MessageKit/pull/697) by [@zhongwuzw](https://github.com/zhongwuzw).
 
+- Added customizable `accessoryView`, with a new `MessagesDisplayDelegate` function `configureAccessoryView`, and corresponding size & padding properties in `MessageSizeCalculator`. [#710](https://github.com/MessageKit/MessageKit/pull/710) by [@hyouuu](https://github.com/hyouuu)
```

Comment:
@hyouuu You will have to pull in the latest changes from `master`. The CHANGELOG is outdated now